templates/commands/goldentest/record_test.go

```diff
@@ -127,6 +127,31 @@ kind: 'GoldenTest'`
 				"test/data/a.txt":         "file A content",
 			},
 		},
+		{
+			name: "failed_to_render_test_case",
+			filesContent: map[string]string{
+				"spec.yaml": `apiVersion: 'cli.abcxyz.dev/v1beta5'
+kind: 'Template'
+
+
```

Comment:
This might be overly complex. Can we just have a spec.yaml whose contents are just `this is not a valid spec file and will fail to parse`?

Reply:
Got it, thanks